Gemfile

```diff
@@ -7,3 +7,7 @@ if facterversion = ENV['FACTER_GEM_VERSION']
 else
   gem 'facter', :require => false
 end
+
+group :dev do
+  gem 'pry'
```

Comment:
Can you add this to the gemspec as a development dependency instead of here?

Reply:
added to gemspec

Reply:
then you can remove it here?